src/body.js

```diff
@@ -0,0 +1,214 @@
+     - With different indentation levels.
+        - That look like this.
+
+
+1. And there are numbered lists too.
+1. Use just 1s if you want!
+1. And last but not least, let's not forget embedded images:
+
+![freeCodeCamp Logo](https://cdn.freecodecamp.org/testable-projects-fcc/images/fcc_secondary.svg)
+`;
+
+const styleOne = {
+  display: 'none',
+};
+
+class Editor extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      textarea: placeholder,
+      editorElementmaximized: false,
+      previewElementMaximized: false,
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleEditorClick = this.handleEditorClick.bind(this);
+    this.handlePreviewClick = this.handleEditorClick.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      textarea: event.target.value,
+    });
+  }
+
+  handleEditorClick() {
+    this.setState({
+      editorElementmaximized: !this.state.editorElementmaximized,
+    });
+    console.log(this.state.editorElementmaximized);
+  }
+
+  handlePreviewClick() {
+    console.log('paola');
+    this.setState({
+      previewElementMaximized: !this.state.previewElementMaximized,
+    });
+  }
+
+  componentDidMount() {
+    document
+      .querySelector('.iconArrow1')
+      .addEventListener('click', this.handleEditorClick);
+    document
+      .querySelector('.iconArrow2')
+      .addEventListener('click', this.handlePreviewClick);
+  }
+
+  render() {
+    if (this.state.editorElementmaximized == true) {
+      document.querySelector('.container').classList.add('minContainer');
+      document.querySelector('.editor').classList.add('maxEditor');
+      document.querySelector('.containerPreview').style.display =
+        styleOne.display;
+    } else if (this.state.previewElementMaximized == true) {
+      document.querySelector('.editor').style.display = styleOne.display;
+    }
+
+    return (
+      <div className="container">
+        <div className="textTareaContainer">
+          <div className="barTextarea">
+            <FontAwesomeIcon
+              className="iconFreeCodeCamp"
+              icon={faFreeCodeCamp}
+            />
+            <p className="pBar">Editor</p>
+            <FontAwesomeIcon className="iconArrow1" icon={faMaximize} />
+          </div>
+          <textarea
+            className="editor"
+            defaultValue={this.state.textarea}
+            onChange={this.handleChange}
+          ></textarea>
+        </div>
+        <Contentwindow textarea={this.state.textarea} />
+      </div>
+    );
+  }
+}
+
+class Contentwindow extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <section className="containerPreview">
+        <div className="barBox">
+          <FontAwesomeIcon className="iconFreeCodeCamp" icon={faFreeCodeCamp} />
+          <p className="pBar">Previewer</p>
+          <FontAwesomeIcon className="iconArrow2" icon={faMaximize} />
```

Comment:
in here we can use the onClick attribute, so it will be something like:
<FontAwesomeIcon className="iconArrow1" icon={faMaximize} onClick={this.this.handleEditorClick} />
Because in React we try to don't interact with the dom directly